process_config.py

```diff
@@ -441,8 +442,8 @@ def parse_args():
 
     parser.add_argument(
         "--output",
-        help=f"folder where DotSlash files should be written, defaults to $GITHUB_WORKSPACE",
-        default=os.getenv("GITHUB_WORKSPACE"),
+        help=f"folder where DotSlash files should be written, defaults to $RUNNER_TEMP",
+        default=os.getenv("RUNNER_TEMP"),
```

Comment:
GITHUB_WORKSPACE is used specifically because it is set when this code is run as a GitHub Action.

Reply:
In my case, there was a folder with the same name as the dotslash file which caused a conflict

See: https://github.com/facebook/dotslash-publish-release/issues/3